### ISSUE TITLE
Legend should be required for ChoiceList

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris-react#1566]
## Issue summary

ChoiceList creates a fieldset and the legend is optional causing accessibility issues.

## Expected behavior

Legend should be required and it can be visually hidden ( already exists )

## Actual behavior

Legends are not needed causing a11y issues.

**Comment by alex-page** [comment]
This is fixed in the version 4.0.0 release
